Updated docs to clarify a few CX questions. (#445)
digital-asset/docs.daml.com: docs/2.8.0/docs/canton/usermanual/upgrading.rst

    .. failure:: participant.start()

The database schema definitions are versioned and hashed. This error informs us about the current
database schema version and how many migrations need to be applied. This check runs on every
startup of the node, so if the node starts, the migrations were successful.

We can now force the migration to a new schema using:

.. snippet:: migrating_participant
    .. success:: participant.db.migrate()

> You can also configure the :ref:`migrations to be applied automatically <migrate_and_start_mode>`.

Reviewer: ```suggestion
You can also :ref:`configure the migrations to be applied automatically <migrate_and_start_mode>`.
```

.. snippet:: migrating_participant
    .. success:: participant.health.ping(participant)

> The ping command involves a create, subsequent exercise and archive of two contracts
> between the admin parties, which means that the full ledger functionality is tested end-to-end.

Reviewer: ```suggestion
The ping command creates two contracts between the admin parties, then exercises and archives them -- providing an end-to-end test of ledger functionality.
```

alt: "serving as an end-to-end test"

Therefore, the protocol upgrade process boils down to:

- Deploy a new domain next to the old domain. Ensure that the new domain is using the desired protocol version.

>   Ensure that you are using different databases (or at least different schemas in the same database)
>   for all the domain nodes (mediator, sequencer node and topology manager), channel names, smart contract addresses etc.
>   It must be a completely separate, new domain (albeit you can reuse your DLT backend as long

Reviewer: ```suggestion
  Also make sure to use different databases (or at least different schemas in the same database)
  for the domain nodes (mediator, sequencer node, and topology manager), channel names, smart contract addresses, etc.
  The new domain must be completely separate, but you can reuse your DLT backend as long
```

>   as you use different sequencer contract addresses or Fabric channels).

Reviewer: ```suggestion
  as you use different sequencer contract addresses or Fabric channels.
```

(apply this change only if you apply the preceding change that removes the open paren)
